File: 100K Coding challenge/4AVERAGE-OF-NUMBERS/src/AVERAGE-OF-NUMBERS.c
/*
 ============================================================================
 Name        : AVERAGE-OF-NUMBERS.c
 Author      : 
 Version     :
 Copyright   : Your copyright notice
 Description : Hello World in C, Ansi-style
 ============================================================================
 */

#include <stdio.h>
#include <stdlib.h>

int main(void) {
	float num1, num2, num3, avg;
	setbuf(stdout, NULL);
	printf("Enter any three numbers to find their average : ");
	scanf("%f%f%f", &num1, &num2, &num3);
	avg = (num1+num2+num3)/3;
	printf("The average of three numbers is : %f", avg);
	return EXIT_SUCCESS;
}

File: 100K Coding challenge/JAVA-100K/AverageOfNumbers.java
import java.util.Scanner;

public class AverageOfNumbers {
    public static void main(String ar[]) {
        Scanner sc = new Scanner(System.in);
        //Main code

        System.out.println("Enter three numbers to find their average:");
        int num1 = sc.nextInt();
        int num2 = sc.nextInt();
        int num3 = sc.nextInt();
        int avg = (num1+num2+num3)/3;
        System.out.println("The average is :"+avg);

        sc.close();
        
    }
    
}
